src/hydra/addParameters.js

```diff
@@ -0,0 +1,41 @@
+import Parameter from "../Parameter";
+import fetchResource from "./fetchResource";
+
+export default api => {
+  const promises = [];
+
+  for (const resource of api.resources) {
+    const promise = fetchResource(resource.url).then(response => {
+      if (!response.parameters) {
+        return [];
+      }
+
+      const resourceParameters = [];
+      const { parameters = {} } = response;
+
+      parameters.forEach(({ property = null, required, variable }) => {
+        if (null === property) {
+          return;
+        }
+
+        const { range = null } = resource.fields.find(
```

Comment:
Is it sure to always match a field here? In my previous comment, I suggested to add a default value (`|| {}`) to avoid runtime errors.